colab/tflite_text_classification.ipynb

```diff
@@ -0,0 +1,514 @@
+        "    label = labels[0]\n",
+        "    confidence = output[0]\n",
+        "  else:\n",
+        "    label = labels[1]\n",
+        "    confidence = output[1]\n",
+        "\n",
+        "  print(\"Label: \" + label + \"\\nConfidence: \" + str(confidence))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "xq00JknF0cOz",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "32ad1ac7-18b4-43fd-f364-a6e1e1d1dab1"
+      },
+      "source": [
+        "#@title Text samples\n",
+        "positive_text = \"This is the best movie I've seen in recent years. Strongly recommend it!\"\n",
+        "negative_text = \"What a waste of my time.\"\n",
+        "\n",
+        "print(positive_text)\n",
+        "print(tokenize_input(positive_text))"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "This is the best movie I've seen in recent years. Strongly recommend it!\n",
+            "[[   1   13    8    3  117   19  206  109   10 1134  152 2301  385   11\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0]]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fvxhY1W7X7Gf"
+      },
+      "source": [
+        "## TFLite\n",
```

Comment:
could qualify this section header (and the "IREE" one below) so people can more easily see that they are independent of one another

```suggestion
        "## Run using TFLite\n",
```